### ISSUE TITLE
Meet and identify data issues to connect sponsoring candidates and leadership PACs

### ISSUE THREAD

**Issue by JonellaCulmer** [fecgov/fec-cms#2369]
**What we're after:** Now that we've identified some issues and potential blockers for connecting leadership PACs to their sponsored candidates, we need to schedule and hold a meeting to go over those potential issues and brainstorm on possible solutions. We should also attempt to outline as many potential problems as possible as part of this discussion. 

From that discussion we should be able to identify what data issues might be blocking this work and mock up possible design solutions to make this information easier to users to find. 

**Completion criteria:** 
- [x] Schedule meeting during 7.2 
- [x] Hold meeting during 7.3
- [x] Compile takeaways and possible next steps
- [x] Mock up design solutions 
- [ ] Document data issues and possible solutions to correct those issues

**Comment by JonellaCulmer** [comment]
Thanks to everyone who participated in the convo this morning!
Here is our list of takeaways and next steps:

### Takeaways:
**Problem:** Sponsors are not consistently provided on form 1. 
**Solution:** The data is there or displayed in some way on committee forms. Experiencing difficulties with form 1S and formatting issues. We need to indentify the root causes before moving forward.

**Problem:** Leadership PAC money is entirely separate from candidate committee money.
**Solution:** To eliminate the problem of confusing users by including summary data or listing all the money spent or raised in association with a particular candidate, we can design for maintaining separate committees and focus on providing users with candidate names. 

### Next steps: 
- [ ] Poke around in the data for form 1S and ensure this piece is functioning for leadership PACs as it should @PaulClark2 
- [ ] Investigate whether the parser or informatica is stopping candidate names from moving forward @PaulClark2 
- [ ] Determine whether this is a DBA problem in which the view that supplies the API is not operating effectively @PaulClark2 
- [ ] Mock up front-end solution to incorporate candidate name filter on All committees data table @JonellaCulmer 
- [ ] Mock up front-end solution to incorporate Sponsor name on leadership PAC pages @JonellaCulmer 
- [ ] Determine whether we can specify the front-end to display sponsor only on committees that are leadership PACs @patphongs 

cc: @AmyKort @wluoFEC @llienfec

**Comment by JonellaCulmer** [comment]
Some preliminary mockups are below for the leadership PAC related pages: All committees datatable and the committee profile pages. Still have some questions that need to be answered by @jwchumley @PaulClark2 and @patphongs before we can really finalize. 

### Designs:
Questions - Are we able to show the same information regarding candidates that we do on candidate committee pages? Is this information available to us if the candidate is only listed as a sponsor?

**Committee profile page:**
<img width="712" alt="screen shot 2018-10-09 at 8 29 19 am" src="https://user-images.githubusercontent.com/31663028/46669513-a5658d00-cb9d-11e8-9e1c-4e361c8beaa2.png">

**All committees data table:**
<img width="323" alt="screen shot 2018-10-09 at 8 29 45 am" src="https://user-images.githubusercontent.com/31663028/46669523-aac2d780-cb9d-11e8-971f-969014e4d6d1.png">

**Comment by llienfec** [comment]
Hey @JonellaCulmer These look really good!  Would there be a way to move the candidate or sponsor name or ID down closer to the committee type?  Or maybe only have it show up as an option when leadership PAC is checked?

**Comment by AmyKort** [comment]
I'm concerned the "candidate or sponsor name" filter is confusing on the all committees data table. Is this looking for Leadership PAC sponsors only--so that "candidate" and "sponsor" are the same thing? Or will it find me authorized committees, JFRs, etc.? Are users familiar with the concept of a "sponsor"? In my experience, they're not.

I'm also a little worried that, with the small number of Leadership PACs we have, this is a really big fix. How hard would it be to implement @llienfec 's idea of having the filter show up only when leadership PAC is selected?

**Comment by JonellaCulmer** [comment]
Thanks @llienfec @AmyKort. We do have the option to only make a filter available once another option is selected, we currently use that on the audit search for category searches. But it was my understanding that regardless of whether the user was specifically interested in a leadership PAC that searching by candidate name or ID. 

Is there another term we might use for sponsor? Maybe authorizing candidate name or ID?
Or does it make more sense to have candidate name or ID, and sponsor separated?

**Comment by AmyKort** [comment]
For me at least it would make more sense to have the candidate name and sponsor separated. Could we have the sponsor option show up only for folks filtering for leadership PACs?

Thanks for your patience @JonellaCulmer !!

**Comment by JonellaCulmer** [comment]
Based on additional feedback and discussions, I've updated the two mockups. 

### Updated filters mockup
The filters for the All committees data table have been updated to just the leadership PAC sponsor search on the Committee type filter. We still need to determine what kind of search this will be. Depending on the data that's available, we will in all likelihood need to limit the sponsor search to a free-text search and not include the type-ahead. Type-aheads are only helpful when you're able to select the specific candidate (with associated ID) and I don't believe we have ID available for our leadership PAC sponsors. Yet. ;)

![screen shot 2018-10-11 at 11 00 15 am](https://user-images.githubusercontent.com/31663028/46813530-e0ee8b80-cd44-11e8-8cf8-ff57a4d7d6f3.png)

### Updated committee profile page mockup
Similar to the changes made for the filters panel, for leadership PAC profile pages, on the About this committee tab, the mockup has been updated to better reflect the available data. At this time, it's unlikely we're able to provide the same information we provide on candidate committee profile pages (i.e. seat, state/district, and party) and that we're limited to providing candidate/officeholder/sponsor name. We may also need to explore alternatives to providing a link to a candidate's profile page. In every other instance on the site, we link a candidate's name to their profile page, and this would be one place where we do not. Through testing, we can explore possible solutions. Particularly if we learn that it is burdensome to users. 

![screen shot 2018-10-11 at 8 03 36 am](https://user-images.githubusercontent.com/31663028/46813946-af29f480-cd45-11e8-99b5-a63643d33b18.png)

cc: @llienfec @AmyKort @jwchumley

**Comment by AmyKort** [comment]
I like it. Is the Leadership PAC sponsor option in the filter grayed out until you select Leadership PAC from the PACs list?

**Comment by JonellaCulmer** [comment]
@AmyKort Thanks for looking at this. Yes, that's the intention.

**Comment by AmyKort** [comment]
Looks great! Thank you so much!

**Comment by llienfec** [comment]
Yep, looks great.  Thanks @JonellaCulmer

**Comment by AmyKort** [comment]
Are the outstanding action items for this ticket all data questions? Should we close this ticket and open a second one about documenting data issues and possible solutions? It seems like @JonellaCulmer 's work is done.

**Comment by JonellaCulmer** [comment]
The design work for this task is complete. Closing in favor of next step to handle the data issues here:  https://github.com/fecgov/openFEC/issues/3433
